testdata/picard/arrays/VerifyIDIntensity.txt

```diff
@@ -3,3 +3,4 @@ ID	%Mix		LLK		LLK0
 0	0.214766	157575	177169
 1	0.214767	157576	177170
 2	0.0994769	90260.4	91166.7
+3   0.234567    -4703.97    -5204.97
```

Comment:
Not sure why this works without tabs? In any event I think it should use tabs instead of spaces.

Reply:
Ugh I have my IDE replace tabs with spaces. It works because the regex uses `\s+`, matching on any whitespace.